### ISSUE TITLE
Headings Alignment fixed 

### ISSUE THREAD

**Issue by HK853** [mindspace-prototype/MindSpace#138]
Heading alignment are fixed move to its right place.
also navbar movement is fix which is improper.

**Comment by hussain123sahil** [comment]
Please assign me this issue.

**Comment by Mandvidangi** [comment]
Please assign me this issue. I would be glad to fix the heading allignment.

**Comment by ASLSwetha** [comment]
I noticed the issue with the alignment of headings. I can definitely help with that!

**Comment by Rahman-Ali** [comment]
Please assign this issue to me, as I would be glad to resolve it.

**Comment by T1rth1** [comment]
yes please assign me this issue..i would like to work upon this issue.
